apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex

```diff
@@ -37,10 +46,19 @@ defmodule OMG.Watcher.ExitProcessor.ExitInfo do
           exiting_txbytes: Transaction.tx_bytes(),
           # this means the exit has been first seen active. If false, it won't be considered harmful
           is_active: boolean(),
-          eth_height: pos_integer()
+          eth_height: pos_integer(),
+          root_chain_txhash: Crypto.hash_t()
         }
 
-  def new(contract_status, %{eth_height: eth_height, call_data: %{output_tx: txbytes}, exit_id: exit_id} = event) do
+  def new(
+        contract_status,
+        %{
```

Comment:
maybe now is a good time to re-approach this pattern matching.

event is a module defined struct, you can access it without pattern matching it in the function header - this way we can reduce the module bloat that was accumulated

Reply:
Done 6287e2b.